Lib/fontmake/font_project.py

```diff
@@ -54,6 +55,7 @@
     GLYPHS_PREFIX + "customParameter.InstanceDescriptorAsGSInstance.Keep Glyphs"
 )
 GLYPH_EXPORT_KEY = GLYPHS_PREFIX + "Glyphs.Export"
+COMPAT_CHECK_KEY = GLYPHS_PREFIX + "customParameter.GSFont.Enforce Compatibility Check"
```

Comment:
how about we also add a CLI option like --compatibility-check or something? I imagine a DS+UFO workflow could take advantage of this as well, and that priavet lib key is pretty obscure